src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement/Services/NationalAchievementRates/Models/NationalAchievementRateBase.cs

```diff
@@ -0,0 +1,7 @@
+namespace SFA.DAS.RoatpCourseManagement.Services.Models
+{
+    public class NationalAchievementRateBase : NationalAchievementRateOverallBase
+    {
```

Comment:
Dont like the base classes at all. Remove all the base classes, it is ok to have common properties explicitly defined. These are two different datasets and must be treated like different entities.